cloud_dashboard/src/organisms/CallbackForm.tsx

```diff
@@ -51,8 +52,19 @@ const CallbackForm = () => {
   }, []);
 
   return <form>
-      <h2>Please waiting...</h2>
-    </form>;
+    <div style={{ display: 'flex' }}>
+      <Loader
+        type="Grid"
+        color="#00BFFF"
+        height="3rem"
+        width="3rem"
+      />
+      <span style={{
+        fontSize: '2rem',
+        marginLeft: '2rem'
+      }}>Orchestrating the clouds...</span>
```

Comment:
@RyoYamashita765 Thank you for the update.  On my second thought, can you change to:

```suggestion
      }}>Orchestrating clouds...</span>
```

Reply:
@naoi I changed the button and the message. Is there any problem with this design?

![スクリーンショット 2021-11-12 15 31 07](https://user-images.githubusercontent.com/48265860/141421087-49277b8b-69f0-4302-9f7b-fb9c6954c91e.png)

![スクリーンショット 2021-11-12 15 31 15](https://user-images.githubusercontent.com/48265860/141421096-09307e05-5728-419c-ab59-14b96e658cd3.png)